src/main/java/com/flansmod/common/eventhandlers/BulletHitEvent.java

```diff
@@ -16,6 +16,7 @@
 import net.minecraft.block.Block;
 import net.minecraft.entity.Entity;
 import net.minecraft.util.MovingObjectPosition;
+import net.minecraft.util.Vec3;
 
 @Cancelable
 public class BulletHitEvent extends Event {
```

Comment:
Would be nice if this event was documented, or at the very least included some JavaDoc comments.